dwave_networkx/drawing/qubit_layout.py

```diff
@@ -143,17 +125,16 @@ def node_color(v):
         # the range of the color map is shared for nodes/edges and is symmetric
         # around 0.
         vmag = max(max(abs(c) for c in node_color), max(abs(c) for c in edge_color))
-        if vmin is None:
-            vmin = -1 * vmag
-
-        if vmax is None:
-            vmax = vmag
 
-        if edge_vmin is None:
-            edge_vmin = -1 * vmag
+        # since we're applying the colormap here, matplotlib throws warnings if
+        # we provide these arguments and it doesn't use them.
+        cmap = kwargs.pop('cmap', plt.get_cmap('coolwarm'))
```

Comment:
Is a change in semantics intended here? Consider the case when user supplies `cmap=None`.

Reply:
The semantics should be the same. In the previous commit, we have
```
if linear_biases:
    cmap = kwargs.pop('cmap', None)
...
if cmap is None:
    cmap = plt.get_cmap('coolwarm')
```
which would have the same behaviour. The issue is if `linear_biases` was never supplied, then the second conditional throws an error as `cmap` is undefined.

Reply:
No, previously if the function was called with `cmap=None` (explicitly set), you would end up with `cmap = plt.get_cmap('coolwarm')` (assuming it doesn't fail).

Now, if user explicitly sets `cmap=None`, the final value will be `cmap = None`.

Reply:
AH, OK I misunderstood.